src/pages/sc-cycle-counts/Forms/Controller.js

```diff
@@ -36,15 +35,15 @@ const Controller = ({ store, maxAccess, labels }) => {
     onSubmit: async () => {
```

Comment:
rows: [
        {
          id: 1,
          controllerId: '',
          siteId: '',
          stockCountId: '',
          status: ''
        }
      ]
      keep it rows: [] to remove initial appearing row